CRM/Core/Payment/Stripe.php

```diff
@@ -669,4 +669,23 @@ function doRecurPayment(&$params, $amount, $stripe_customer) {
   function doTransferCheckout(&$params, $component) {
     CRM_Core_Error::fatal(ts('Use direct billing instead of Transfer method.'));
   }
+
+  /**
+   * Stripe payment instrument validation.
```

Comment:
Let's make the description imperative: "Validates Stripe payment instrument"
Also, if you could provide a link to the payment instrument like that:
`@see https://stripe.com/some-documentation-page` that would be great for devs.